cloud/k8s/deploy/k8s-deploy.sh

```diff
@@ -67,8 +67,12 @@ function configure_cri_runtime() {
 		configure_containerd
 		;;
 	esac
-	[ -c /host-devices/kvm ] && chmod 666 /host-devices/kvm
-	[ -c /host-devices/kkm ] && chmod 666 /host-devices/kkm
+
+        [ -c /host-dev/kvm ] && chmod 666 /host-dev/kvm
+        mkdir -p /etc/udev/rules.d
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666"' | tee /etc/udev/rules.d/99-perm.rules
+        udevadm control --reload-rules && udevadm trigger
```

Comment:
This runs in the container, I wonder if does what we need to happen. IOW which udev does it kick?

Reply:
The YAMP file maps the node's `/var/run/dbus` and `/run/system` into the DaemonSet container. The reason for this is so things like `systemctl` and `udevadm` run from the DamonSet end up talking to the Node's daemons.

Reply:
`udevadm control --reload-rules` will reload this container's udev, not the host's, which means the host's one will not know of the change

Reply:
Latest commit maps udev socket so udevadm talks to Node daemon.